src/_P023_OLED.ino

```diff
@@ -13,10 +13,15 @@
 #define PLUGIN_ID_023         23
 #define PLUGIN_NAME_023       "Display - OLED SSD1306"
 #define PLUGIN_VALUENAME1_023 "OLED"
+#define PLUGIN_023_MAX_DYSPALY 2
 
-byte Plugin_023_OLED_address = 0x3c;
-byte Plugin_023_OLED_type = 0;
-byte Plugin_023_OLED_font_width = 0;
+struct Plugin_023_OLED_SettingStruct
```

Comment:
No initialization of the members of this struct.
Its values will be used every second and thus may affect performance of the entire system if the plugin is not used (and thus no init was called)

Reply:
When not explicitly initialized does not use dafult values?